src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 // Relative imports.
 import * as customPropTypes from '../prop-types';
-
+import { findFormsMomentDateFormat } from '../constants';
```

Comment:
Could we remove the verb from this constant? Otherwise it sounds like this is a function that will help you find the moment date format -- maybe we could do FORM_MOMENT_DATE_FORMAT or something similar? 🙂